constrained-voucher.mkd

```diff
@@ -210,7 +212,7 @@ following more complete response is possible.
 # BRSKI-EST protocol {#brski-est}
 
 The constrained BRSKI-EST protocol described below is between the Pledge and the Registrar only.
-(probably via an enrollment proxy, such as described in {{?I-D.ietf-anima-constrained-join-proxy}})
+(probably via a join proxy, such as described in {{?I-D.ietf-anima-constrained-join-proxy}})
```

Comment:
In the RPL space, the use of the term "join" has caused a lot of confusion, as nodes join an RPL mesh (after enrollment).  No amount of clarification had helped, but changing terms did.